From a7673ab3ebaaf9bdb6bd840791c1ad672ab7f341 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 29 Jan 2014 13:36:37 +0000
Subject: [PATCH] Don't include linux/if_bridge.h

Instead rely on struct ethhdr from net/ethernet.h

This fixes build error with musl libc:
In file included from /usr/include/linux/if_bridge.h:17:0,
                 from network.c:47:
/usr/include/linux/if_ether.h:133:8: error: redefinition of 'struct ethhdr'
 struct ethhdr {
        ^
In file included from /usr/include/net/ethernet.h:10:0,
                 from network.c:42:
/usr/include/netinet/if_ether.h:93:8: note: originally defined here
 struct ethhdr {
        ^

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 src/lxc/network.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/lxc/network.c b/src/lxc/network.c
index bd2a37c..34845d1 100644
--- a/src/lxc/network.c
+++ b/src/lxc/network.c
@@ -44,7 +44,6 @@
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
 #include <linux/sockios.h>
-#include <linux/if_bridge.h>
 
 #include "nl.h"
 #include "network.h"
-- 
1.8.5.3

